### PR TITLE
[Refactoring] Tips and tricks refactoring

### PR DESCRIPTION
**Pull request by Benerous**
Breakpoint: 1440px
image

Breakpoint: 1024px
image

Breakpoint: 425px
image

### CODE REVIEW

**Review by skrebetsnataliia — APPROVED** [review]

**Review by luchward — APPROVED** [review]

**Review by vitaliipolitylo — APPROVED** [review]

**Review by romanovskyroma — APPROVED** [review]

File: src/app/component/home/components/useful-tips/tips-list/tips-list.component.ts [hunk 1]
```diff
@@ -52,5 +58,11 @@ export class TipsListComponent implements OnInit {
 
   constructor() { }
 
-  ngOnInit() { }
+  ngOnInit() {
+    this.currentTip = this.tips[0];
+  }
+
+  public changeTip(): void {
```
Comment (JimHolman1199) [inline src/app/component/home/components/useful-tips/tips-list/tips-list.component.ts:65]: empty function?

**Review by JimHolman1199 — APPROVED** [review]

File: src/app/component/comments/components/comment-pagination/comment-pagination.component.html [hunk 2]
```diff
@@ -4,7 +4,8 @@
                        [maxSize]="calcPaginationSize(p.getLastPage(), p.getCurrent())"
                        (pageChange)="onPageChange($event)"
                        >
-    <div class="custom-pagination">
+    <div class="custom-pagination"
+         *ngIf='p.getTotalItems() > 10'>
```
Comment (mr-pavlenko) [inline src/app/component/comments/components/comment-pagination/comment-pagination.component.html:8]: Magic number 10?

File: src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.spec.ts [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import { TranslateModule } from '@ngx-translate/core';
+import { async, ComponentFixture, TestBed, inject } from '@angular/core/testing';
+
+import { EcoNewsWidgetComponent } from './eco-news-widget.component';
+import { RouterTestingModule } from '@angular/router/testing';
+import { NewsListGalleryViewComponent } from '../../news-list/news-list-gallery-view/news-list-gallery-view.component';
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { of } from 'rxjs';
+import { EcoNewsService } from '@eco-news-service/eco-news.service';
```
Comment (mr-pavlenko) [inline src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.spec.ts:9]: Real service

File: src/app/component/home/components/useful-tips/tips-card/tips-card.component.scss [hunk 4]
```diff
@@ -1,31 +1,30 @@
...
-  font-family: 'Open Sans', sans-serif;
-  font-size: 18px;
-  font-weight: normal;
-  font-stretch: normal;
-  font-style: normal;
-  line-height: 1.5;
-  letter-spacing: normal;
+  p {
+    margin: 0 auto;
+    color: #324148;
```
Comment (mr-pavlenko) [inline src/app/component/home/components/useful-tips/tips-card/tips-card.component.scss:21]: use variable

File: src/app/component/comments/components/comment-pagination/comment-pagination.component.html (hunk 2 above)
Comment (skrebetsnataliia) [reply src/app/component/comments/components/comment-pagination/comment-pagination.component.html:8]: It's my mistake. I've fixed it  in this PR https://github.com/ita-social-projects/GreenCityClient/pull/685.

File: src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.spec.ts (hunk 3 above)
Comment (JimHolman1199) [reply src/app/component/eco-news/components/eco-news-detail/eco-news-widget/eco-news-widget.component.spec.ts:9]: fixed here https://github.com/ita-social-projects/GreenCityClient/pull/690

**Pull request merged**